src/App.jsx

```diff
@@ -1,25 +1,61 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import "./App.css";
 
-function City(props) {
-  return <div>This is the City component</div>;
-}
+import City from "./components/City";
+
+const BASE_URL = "https://ctp-zip-code-api.onrender.com/";
 
-function ZipSearchField(props) {
-  return <div>This is the ZipSearchField component</div>;
+async function fetchZipData(zipField) {
+  try {
+    const response = await fetch(BASE_URL + "zip/" + zipField);
```

Comment:
```suggestion
    const response = await fetch(`${BASE_URL}zip/${zipField});
```

This is another way to write what you wrote and may read better. Not that your way is wrong. Just template literals are more expected these days.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals